code/modules/mob/living/carbon/human/human_attackhand.dm

```diff
@@ -151,8 +151,7 @@
 				var/armor_block = run_armor_check(affecting, "melee")
 
 				if(HULK in M.mutations)
-					damage += 5
-					Weaken(4)
+					adjustBruteLoss(15)

```

Comment:
This is bruteloss for a reason, as opposed to + damage; increasing the damage would actually increase the chance of them to stun over a regular human (which we don't want); likewise, this means that that 15 brute will be applied *over* their entire body instead of a specific organ (which can result in lucky strikes of 2 punches = both their hands are broken).